pkg/seccomp/seccomp_test.go

```diff
@@ -0,0 +1,211 @@
+// +build seccomp
```

Comment:
cut/paste  and/or leftover debug cruft?

Reply:
No this change was intended since the test should only run when seccomp is available on the system.